eth_hash/backends/__init__.py

```diff
@@ -9,6 +9,6 @@
 """
 
 SUPPORTED_BACKENDS = [
-    "pysha3",
     "pycryptodome",
+    "pysha3",
```

Comment:
If this is the change that makes pycryptodome the default, maybe add a comment above `SUPPORTED_BACKENDS` that the first is the default. It's not otherwise obvious from a local reading.

Reply:
It's just for the error message. Now it looks like:
```
ImportError: None of these hashing backends are installed: ['pycryptodome', 'pysha3'].
Install with `pip install eth-hash[pycryptodome]`.
```